src/main/java/com/google/error_analyzer/data/Document.java

```diff
@@ -47,4 +48,23 @@ public String getID() {
         return id;
     }
 
+    public boolean equals(Document other) {
+        return (other.getID().equals(this.id) && other.getJsonString().equals(this.jsonString));
+    }
+
+    public static Boolean compareDocumentList(List < Document > expected, List < Document > actual) {
```

Comment:
Where is this getting used?

Reply:
FilterErrorsTest

Reply:
Removed the function and changed Document.equals() function